lib/settings/about_screen.dart

```diff
@@ -16,185 +16,248 @@
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-// ignore_for_file: avoid_print
-// ignore_for_file: library_private_types_in_public_api
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter/scheduler.dart';
+import 'package:logging/logging.dart';
 import 'package:orion/pubspec.dart';
+import 'package:orion/themes/themes.dart';
+import 'package:orion/util/orion_config.dart';
 import 'package:qr_flutter/qr_flutter.dart';
+import 'package:toastification/toastification.dart';
+import 'dart:io';
+
+Logger _logger = Logger('AboutScreen');
+
+Future<String> executeCommand(String command, List<String> arguments) async {
+  final result = await Process.run(command, arguments);
+  if (result.exitCode == 0) {
+    return result.stdout.trim();
+  } else {
+    throw Exception(
+        'Failed to execute command: $command ${arguments.join(" ")}\nError: ${result.stderr}');
+  }
+}
+
+Future<String> getRaspberryPiModel() async {
+  try {
+    final model = await executeCommand('cat', ['/proc/device-tree/model']);
+    _logger.info('Raspberry Pi model: $model');
+    return model.trim();
+  } catch (e) {
+    _logger.warning('Error getting Raspberry Pi model: $e');
+    return 'Unknown Model';
+  }
+}
 
 Future<String> getVersionNumber() async {
-  return Pubspec.version;
+  return 'Orion ${Pubspec.version}' ' - Odyssey 1.0.0';
```

Comment:
Should we toss a TODO in here for using the odyssey endpoint when it exists? Maybe even stub out the api_services method, and have that return the hardcoded odyssey version?

Reply:
Agreed. Resolved by [811e9fc](https://github.com/TheContrappostoShop/Orion/pull/14/commits/811e9fca935c91187b64a20e65b130c7424ae331) for now